<ui version="4.0" >
 <class>TaskSettings</class>
 <widget class="QDialog" name="TaskSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Groups</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLineEdit" name="lineGroup" />
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd" >
          <property name="text" >
           <string>Add</string>
          </property>
          <property name="default" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QListWidget" name="lstGroups" />
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QPushButton" name="btnUp" >
            <property name="text" >
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnDown" >
            <property name="text" >
             <string>Down</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove" >
        <property name="text" >
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineGroup</sender>
   <signal>returnPressed()</signal>
   <receiver>btnAdd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>260</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>333</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
